### PR TITLE
Feature/14 create movie adapter

### PR DESCRIPTION
**Pull request by fppuma**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.android.application'
     id 'kotlin-android'
     id 'kotlin-parcelize'
+
```
Comment (JaimeBallesta) [inline app/build.gradle]: why has this unnecessary space?

File: app/build.gradle [hunk 2]
```diff
@@ -17,6 +18,9 @@ android {
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+    buildFeatures {
+        viewBinding true
+    }
```
Comment (cmauroie) [inline app/build.gradle]: This already exists on build.gradle

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
\ No newline at end of file
+        <fragment android:name="androidx.navigation.fragment.NavHostFragment"
+            android:id="@+id/fragContent"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:navGraph="@navigation/nav_graph"
+            app:defaultNavHost="true"/>
+
+    </RelativeLayout>
+</androidx.coordinatorlayout.widget.CoordinatorLayout>
```
Comment (cmauroie) [inline app/src/main/res/layout/activity_main.xml]: this layout is not necesary update

File: app/build.gradle [hunk 4]
```diff
@@ -52,11 +56,17 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
 
     implementation 'com.github.bumptech.glide:glide:4.12.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.3.4'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.3.4'
```
Comment (cmauroie) [inline app/build.gradle]: currently not used in the project

**Review by cmauroie — CHANGES_REQUESTED** [review]
The solution is not based on the 01-init tag

**Review by juanncode — COMMENTED** [review]
Review tag 01 please

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (fppuma) [reply app/src/main/res/layout/activity_main.xml]: restored to previous version

File: app/build.gradle (hunk 4 above)
Comment (fppuma) [reply app/build.gradle]: restored to previous version

**Review by cmauroie — APPROVED** [review]
perfect

**Review by juanncode — APPROVED** [review]

**Review by JaimeBallesta — APPROVED** [review]

**Pull request merged**